catalog/core/catalog-core-impl/pom.xml

```diff
@@ -22,6 +22,12 @@
     <artifactId>catalog-core-impl</artifactId>
     <name>DDF :: Catalog :: Core :: Impl</name>
     <packaging>pom</packaging>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-math3</artifactId>
```

Comment:
❓ You're including the dependency here, in a `pom` artifact, when you're using it in the `filter-proxy` sub-module. But `filter-proxy` is a bundle and you'll either need to embed this or get it picked up from the OSGi container. The only other place we're currently using `commons-math3` (at version 2.2) is in `metrics-reporting` where it is embedded. Furthermore, I see no place where we wrap the `commons-math3` module.

So you'll compile and unit tests will pass, but at runtime you should be seeing a failure to install for a large number of bundles.

I would recommend wrapping `commons-math3` and installing it pretty early in the installation process. Then remove it as an embedded bundle from `filter-proxy` so it can import the package from the container.